src/webgl/material.js

```diff
@@ -1223,10 +1223,10 @@ p5.RendererGL.prototype._applyColorBlend = function (colors) {
 /**
  * @private sets blending in gl context to curBlendMode
  * @param  {Number[]} color [description]
- * @return {Number[]]}  Normalized numbers array
+ * @return {Number[]}  Normalized numbers array
  */
 p5.RendererGL.prototype._applyBlendMode = function () {
-  if (this._cachedBlendMode === this.curBlendMode) {
+  if (this.curBlendMode === this.preEraseBlend) {
```

Comment:
Thinking this through a bit more, I think this might be slightly mixing the logic a bit. I think this method's intention is to only change the blend settings if the mode has changed from what we were last using, regardless of whether or not that was used for erasing. So I believe we need two stored properties: a `preEraseBlend` like you have, but also a `cachedBlendMode`, referring to the last value that `_applyBlendMode` was called with. So everything in this method would be using a `cachedBlendMode`, and the `erase`/`noErase` ones would refer to `preEraseBlend`.

Reply:
Thanks @davepagurek for the insights. I'll just look into this and get back to you soon ..!